docs/concepts/json_schema.md

```diff
@@ -481,6 +482,50 @@ print(json.dumps(Foo.model_json_schema(), indent=2))
 """
 ```
 
+### Programmatic field title generation
+
+The `field_title_generator` parameter can be used to programmatically generate the title for a field based on its name.
+
+See the following example:
+
+```py
+import json
+
+from pydantic import BaseModel, Field
+
+
+def make_title(field_name: str) -> str:
```

Comment:
The full field as in the `FieldInfo` object? That we need to pass both `FieldInfo` and the name of the field since `FieldInfo` doesn't include the name.

Then the signature of `make_title` would be 

```python
def make_title(field_name: str, field_info: FieldInfo) -> str: ...
```